platform/src/main/resources/anchor-config-default-values.yaml

```
@@ -0,0 +1,456 @@
+####################################################################################################
+## Anchor Platform - Default Configuration
+##
+## Values not defined in the user config use the default values defined here.
+####################################################################################################
+
+version: 0.2
```

Can we use integer number? 
```suggestion
version: 1
```

---

Sounds good